src/lib/components/workflow/pending-activity/workflow-pending-activity.svelte

```diff
@@ -0,0 +1,220 @@
+  const { namespace, workflow, run } = $derived(page.params);
+
+  const failed = $derived(activity.attempt > 1);
+  const scheduledEvent = $derived(
+    $fullEventHistory?.find(
+      (event) =>
+        isActivityTaskScheduledEvent(event) &&
+        event.attributes.activityId === activity.activityId,
+    ),
+  );
+</script>
+
+<Card
+  class="flex flex-col items-start gap-2 overflow-hidden bg-primary md:flex-row"
+>
+  <div class="flex flex-1 flex-col space-y-2 truncate">
+    <div class="flex-1">
+      <div class="flex flex-wrap items-center space-x-3">
+        <WorkflowStatus status={activity.paused ? 'Paused' : activity.state} />
+        <h4>{activity.activityType}</h4>
+      </div>
+    </div>
+    <div class="grid grid-cols-1 gap-2 xl:grid-cols-2">
+      {@render detail(translate('workflows.activity-id'), activity.activityId)}
+      {@render detail(translate('workflows.attempt'), attempts)}
+      {#if activity.scheduledTime}
+        {@render detail(translate('workflows.next-retry'), nextRetry)}
+      {/if}
+      {#if activity.expirationTime}
+        {@render detail(
+          translate('workflows.retry-expiration'),
+          formatRetryExpiration(
+            activity.maximumAttempts,
+            formatDuration(
+              getDuration({
+                start: Date.now(),
+                end: activity.expirationTime,
+              }),
+            ),
+          ),
+        )}
+      {/if}
+      {#if activity.lastHeartbeatTime}
+        {@render detail(
+          translate('workflows.last-heartbeat'),
+          formatDate(activity.lastHeartbeatTime, $timeFormat, {
+            relative: $relativeTime,
+            relativeStrict: true,
+          }),
+        )}
+      {/if}
+      {#if activity.lastAttemptCompleteTime}
+        {@render detail(
+          translate('workflows.last-attempt-completed-time'),
+          formatDate(activity.lastAttemptCompleteTime, $timeFormat, {
+            relative: $relativeTime,
+          }),
+        )}
+      {/if}
+      {#if activity.lastStartedTime}
+        {@render detail(
+          translate('workflows.last-started-time'),
+          formatDate(activity.lastStartedTime, $timeFormat, {
+            relative: $relativeTime,
+          }),
+        )}
+      {/if}
+      {@render detail(
+        translate('workflows.scheduled-event'),
+        scheduleEventLink,
+      )}
+      {#if activity.lastWorkerIdentity}
+        {@render detail(
+          translate('workflows.last-worker-identity'),
+          activity.lastWorkerIdentity,
+        )}
+      {/if}
+    </div>
+  </div>
+  <div class="flex w-full flex-col gap-2 md:flex-1">
+    {#if failed}
+      {@render failures()}
+    {/if}
+    {#if activity.heartbeatDetails}
+      {@render detail(translate('workflows.heartbeat-details'), heartbeat)}
+    {/if}
+  </div>
+</Card>
+
+{#snippet detail(label: string, value: string | number | Snippet)}
+  <div>
+    <p class="text-secondary">{label}</p>
```

Comment:
How would you feel about `font-mono` here (similar to what we have for the workflow summary)?
```suggestion
    <p class="font-mono">{label}</p>
```

Reply:
I went with font-mono text-sm. What I want to do after this is show this same card on the expanded event summary row and make all the activity details into cards

<img width="1807" alt="Screenshot 2025-05-01 at 11 54 37 AM" src="https://github.com/user-attachments/assets/cca51f85-4bb6-4cea-a147-b9d97d360b79" />